service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/ValidateOrderController.java

```diff
@@ -53,6 +53,25 @@ public AboutToStartOrSubmitCallbackResponse handleMidEventValidateAddress(
             .build();
     }
 
+    @PostMapping("/children/mid-event")
+    public AboutToStartOrSubmitCallbackResponse handleMidEventValidateChildren(
+        @RequestBody CallbackRequest callbackRequest) {
+
+        final CaseDetails caseDetails = callbackRequest.getCaseDetails();
+        final CaseData caseData = mapper.convertValue(caseDetails.getData(), CaseData.class);
+
+        List<String> errors = List.of();
```

Comment:
emptyList() ?

Reply:
Defaults to immutable list, changed to ArrayList